### PR TITLE
[Django upgrade] RemovedInDjango40Warning: request.is_ajax() is deprecated in Django3.1

### PR DESCRIPTION
**Pull request by agl29**
## What changes were proposed in this pull request?

- request.is_ajax() -> request.headers.get('x-requested-with') == 'XMLHttpRequest'

### CODE REVIEW

**Comment by agl29** [comment]
usefull links 
1 -> https://docs.djangoproject.com/en/3.2/releases/3.1/#id2
2 -> https://docs.djangoproject.com/en/3.2/ref/request-response/#django.http.HttpRequest.is_ajax

**Review by romainr — APPROVED** [review]

**Comment by agl29** [comment]
rereview @romainr 
earlier one was giving error with django1.11.
because request.headers is new in django2.2.
https://docs.djangoproject.com/en/2.2/ref/request-response/#django.http.HttpRequest.headers

File: apps/filebrowser/src/filebrowser/views.py [hunk 1]
```diff
@@ -250,6 +250,11 @@ def view(request, path):
           '/filebrowser/view=' + urllib_quote(trash_path.encode('utf-8'), safe=SAFE_CHARACTERS_URI_COMPONENTS)
       )
 
+  if sys.version_info[0] > 2:
+    _is_ajax = request.headers.get('x-requested-with') == 'XMLHttpRequest'
+  else:
+    _is_ajax = request.is_ajax()
+
```
Comment (romainr) [inline apps/filebrowser/src/filebrowser/views.py:257]: Let's not pollute the code too much this way, how about adding a `is_ajax(request)` function in `desktop/core/src/desktop/lib/view_util.py` and just call it later?

Comment (agl29) [reply apps/filebrowser/src/filebrowser/views.py]: done @romainr

**Review by romainr — APPROVED** [review]
Nice & Sweet!

(minus the no test :))

**Pull request merged**